src/dotty/tools/dotc/printing/Formatting.scala

```diff
@@ -0,0 +1,175 @@
+package dotty.tools.dotc
+package printing
+
+import core._
+import Texts._, Types._, Flags._, Names._, Symbols._, NameOps._, Contexts._
+import collection.mutable
+import collection.Map
+import Decorators._
+import scala.annotation.switch
+import scala.util.control.NonFatal
+import reporting.Diagnostic
+
+object Formatting {
+
```

Comment:
Decorators acts like an extended Predef for the dotty compiler. Since string interpolators are ubiquitous I think they should be part of it, and not require a separate import.